csharp/HOLMS.Platform/HOLMS.Platform/Support/Currency/DollarCents.cs

```diff
@@ -34,11 +35,11 @@ public struct DollarCents : IComparable<DollarCents>, IEquatable<DollarCents> {
 
         public static DollarCents Zero => new DollarCents(false, 0, 0);
 
-        public int TotalCents => (int)Math.Abs(Dollars * 100 + Cents);
+        public int TotalCents => (int)((IsNegative ? -1 : 1) * (Dollars * 100 + Cents));
```

Comment:
Tch...I'm not a fan of inlining a literal :100: for the dollars to cents conversion. It should be a constant "CentsPerDollar" :trollface: